src/containers/AddPost/AddPost.js

```diff
@@ -3,14 +3,34 @@ import TextField from "@material-ui/core/TextField/TextField";
 import Button from "@material-ui/core/Button";
 import Navigations from "../../components/Navigations/Navigations";
 import classes from './AddPost.module.css';
+import * as actions from '../../store/actions/index';
+import {connect} from 'react-redux';
+
+let title = '';
+let description = '';
 
 class AddPost extends Component {
+
+    addPostHandler = () => {
+        this.props.onAddPost(title, description);
+    }
+
+    titleHandler = (event) => {
+        event.preventDefault();
+        title = event.target.value;
+    }
+
+    descriptionHandler = (event) => {
+        event.preventDefault();
+        description = event.target.value;
+    }
+
     render() {
         return (
             <div>
                 <Navigations/>
                 <form className={classes.FormClass}>
-                    <TextField id="standard-secondary" label="Title" color="secondary" />
+                    <TextField id="standard-secondary" label="Title" color="secondary" onChange={(event) => this.titleHandler(event)}/>
```

Comment:
onChange={(event) => this.titleHandler(event)} 
it is the same as 
onChange={this.titleHandler}
right?